games/cmatrix/Portfile

```diff
@@ -11,32 +11,31 @@ description			Console Matrix
 long_description	Console Matrix simulates the display from "The Matrix"
 homepage			http://www.asty.org/cmatrix/
 
-master_sites		${homepage}dist
-checksums			md5 ebfb5733104a258173a9ccf2669968a1 \
-					sha1 ca078c10322a47e327f07a44c9a42b52eab5ad93 \
-					rmd160 8a401753ca0e88c0d24b6a8c3f23b725368c13bb
+checksums			rmd160  73e38f7f1e903a738e98724bcc326b4c71034b44 \
+					sha256  456d8097cb4724a69337e9803dd9b7079fa10684d22e35fc2bdd88655c03925a \
+					size    205672
 
+depends_build       port:autoconf port:automake
 depends_lib         port:ncurses
 
-patchfiles          Makefile.in.patch
-
+configure.cmd       autoreconf -i && ./configure
```

Comment:
you probably want to use `use_autoreconf` here, see [the guide](https://guide.macports.org/#reference.phases.configure.automake-autoconf)